internal/lyric/lrc.go

```diff
@@ -21,21 +21,28 @@ type LRCFile struct {
 	fragments []LRCFragment
 }
 
-func (f *LRCFile) AsText() string {
+func AsText(f *LRCFile, t *TranslateLRCFile) string {
 	var builder strings.Builder
 
 	if f == nil || len(f.fragments) == 0 {
-		return "暂无歌词~"
+		return "[00:00.00]暂无歌词~"
 	}
 
 	for _, line := range f.fragments {
+		flt := time.Duration(line.StartTimeMs*1e6) % (time.Second * 60)
+		it := time.Duration(line.StartTimeMs * 1e6)
+		builder.WriteString(fmt.Sprintf("[%02.0f:%05.2f]", it.Minutes(), flt.Seconds()))
```

Comment:
我在尝试利用这个功能制作桌面歌词，个人感觉这里直接记录 `StartTimeMs` 会更方便一些，更容易和 `org.mpris.MediaPlayer2.Player.Position` 进行对比，不需要进行额外的解析和转换。
```suggestion
		builder.WriteString(fmt.Sprintf("[%08d]", line.StartTimeMs))
```

Reply:
这个是lrc的标准格式，是提供给lrc解析器使用的

Reply:
我知道这个，主要是我用这个 PR，发现会出现如下情况。歌词的顺序是对的，但分钟数会出问题（似乎 `it.Minutes()` 会把 90s 输出为 2min）。所以直接改成了毫秒数，不会出错。
![image](https://github.com/go-musicfox/go-musicfox/assets/26739999/6f7f5bbf-5b04-4f94-9302-4e7ba5abdc7d)

Reply:
[3049fbc](https://github.com/go-musicfox/go-musicfox/pull/216/commits/3049fbc1c8a0b9cfe983924b0a3ea9cc947b1434)